requirements.txt

```diff
@@ -50,7 +50,7 @@ flake8>=6.1.0
 fqdn>=1.5.1
 frozenlist>=1.4.0
 fsspec>=2023.6.0
-git-python>=3.1.0
+GitPython>=3.1.0
```

Comment:
I think the package changed its name on pypi recently, thank you for seeing this!

Reply:
there is a git-python on pypi but no version 3.1.0, which pipven happily complaint about

Reply:
Yes I checked and it's indeed the same package, the authors probably decided to rename it and discontinue the distribution undeer the git-python name. 
Very helpful thank's!